src/observable/GenerateObservable.ts

```diff
@@ -245,7 +245,7 @@ export class GenerateObservable<T, S> extends Observable<T> {
     } while (true);
   }
 
-  private static dispatch<T, S>(state: SchedulerState<T, S>) {
+  private static dispatch<T, S>(state: SchedulerState<T, S>): Subscription | void {
```

Comment:
Yes, I know. just thought of reusing existing types if possible. It's too complex to introduce another type for this, maybe ok as is.